inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/entity/InLongTopic.java

```diff
@@ -26,6 +27,7 @@
     private int partitionId;
     //pulsar,kafka,tube
     private String topicType;
+    private Map<String, Object> properties;
```

Comment:
Map<String, Object> properties
It is not fit between variable name(properties) and type.
java.util.Properties is better for variable name(properties) .